pmm-app/tests/upgradePMM_test.js

```diff
@@ -4,92 +4,132 @@ const serviceNames = {
   postgresql: 'postgres_upgrade_service',
   mongodb: 'mongodb_upgrade_service',
   proxysql: 'proxysql_upgrade_service',
-  rds: 'mysql_rds_uprgade_service'
+  rds: 'mysql_rds_uprgade_service',
 };
 
 Feature('PMM server Upgrade');
 
-Before(async (I) => {
+Before(async I => {
   I.Authorize();
   I.setRequestTimeout(30000);
 });
 
 Scenario(
-    'Verify user can create Remote Instances before upgrade and they are in RUNNNING status @pmm-upgrade @visual-test @not-pr-pipeline',
-    async (I, homePage, inventoryAPI, addInstanceAPI) => {
-      // Adding instances for monitoring
-      for (const type of Object.values(addInstanceAPI.instanceTypes)) {
-        await addInstanceAPI.apiAddInstance(type, serviceNames[type.toLowerCase()]);
-      }
-      // Checking that instances are RUNNING
-      for (const service of Object.values(inventoryAPI.services)) {
-        await inventoryAPI.verifyServiceExistsAndHasRunningStatus(service,
-            serviceNames[service.service]);
-      }
+  'PMM-T289 Verify Whats New link is presented on Update Widget @pmm-upgrade @visual-test @not-pr-pipeline',
+  async (I, homePage) => {
+    I.amOnPage(homePage.url);
+    const res2 = process.env.DOCKER_VERSION.split('.');
+    const res = process.env.PMM_SERVER_LATEST.split('.');
+    const majorVersionDiff = res[1] - res2[1];
+    const patchVersionDiff = res[2] - res2[2];
+    if (majorVersionDiff >= 1 && patchVersionDiff >= 0) {
+      I.waitForElement(homePage.fields.whatsNewLink, 30);
+      I.seeElement(homePage.fields.whatsNewLink);
+      const link = await I.grabAttributeFrom(homePage.fields.whatsNewLink, 'href');
+      assert.equal(link.indexOf('https://per.co.na/pmm/') > -1, true, 'Whats New Link has an unexpected URL');
     }
+  }
 );
 
 Scenario(
-    'Verify user is able to Upgrade PMM version @pmm-upgrade @visual-test @not-pr-pipeline',
-    async (I, inventoryAPI, homePage) => {
-      I.amOnPage(homePage.url);
-      await homePage.upgradePMM();
-    }
+  'PMM-T288 Verify user can see Update widget before upgrade @visual-test @not-pr-pipeline',
+  async (I, adminPage, homePage) => {
+    I.amOnPage(homePage.url);
+    await homePage.verifyPreUpdateWidgetIsPresent();
+  }
 );
 
 Scenario(
-    'Verify Agents are RUNNING after Upgrade (API) @pmm-upgrade @visual-test @not-pr-pipeline',
-    async (I, inventoryAPI) => {
-      for (const service of Object.values(inventoryAPI.services)) {
-        await inventoryAPI.verifyServiceExistsAndHasRunningStatus(service,
-            serviceNames[service.service]);
-      }
+  'Verify user can create Remote Instances before upgrade and they are in RUNNNING status @pmm-upgrade @visual-test @not-pr-pipeline',
```

Comment:
This may be unrelated to testing PMM Upgrade functionality

Reply:
Well for us we define what we are doing in the Feature and for us, this test file Feature is about testing Upgrade and upgrade related test cases from our test cycle. 
`Feature('PMM server Upgrade Tests and Executing test cases related to Upgrade Testing Cycle');` 
I have modified the Feature with more detail to make it clear.

Reply:
Got it, it's clear now with that description :)